docs/documents/query.mdx

```diff
@@ -1,53 +1,125 @@
 ---
 title: "Query documents"
-description: "Use the `query` method to search for documents."
+description: "Use the `query` function to search for documents."
 ---
 
 import Support from "/snippets/support.mdx";
 
-TopK allows you to query documents using a SQL-inspired syntax. It comes with **semantic search**, **text search**, **vector search**, and **filtering capabilities** out of the box.
+TopK provides a SQL-inspired syntax for querying documents. It features built-in **semantic search**, **text search**, **vector search**, metadata **filtering** as well as **reranking** capabilities.
```

Comment:
instead of `SQL-inspired` we can call it "data frame-like" so people know its similar to numpy/etc

Reply:
this also reminds me, tpuf and other have helper methods to convert their result collection into numpy array directly. I wouldn't pollute our api with this bs, but we can mention/show it in docs

Reply:
see numpy/pandas/polars for py, and danfo.js/dataframe-js for js. imo "dataframe api" is a common abstraction for people working with data